.eslintrc

```diff
@@ -1,14 +1,16 @@
 {
   "rules": {
     // This project uses `console.log()`.
-    "no-console": "off"
+    "no-console": "off",
+    "amo/i18n-no-tagged-templates": "error",
   },
   "env": {
     "node": true,
     "browser": true,
     "es6": true,
   },
-  "extends": ["amo/base"],
+  "plugins": ["amo"],
+  "extends": ["amo"],
```

Comment:
That includes a bunch of react rules addons-linter doesn't need though, right ? So why that change ? Is it just for consistency across our projects ?

Reply:
ah, right. That's a mistake...